build_deps/toolchains/hkv/hkv.BUILD

```diff
@@ -0,0 +1,18 @@
+load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda", "if_cuda_is_configured")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "hkv",
+    hdrs = glob([
+        "include/merlin/core_kernels/*.cuh",
+        "include/merlin/*.cuh",
+        "include/merlin_hashtable.cuh",
+        "include/merlin_localfile.hpp",
```

Comment:
Considering the possible addition of including files in the top-level directory in the future, I recommend this:
```       
        "include/*.cuh",
        "include/*.hpp",
```

Reply:
DONE